### ISSUE TITLE
"Dropdown with keyboard nav example" yields "Uncaught TypeError: Cannot read properties of undefined (reading 'focus')"

### ISSUE THREAD

**Issue by kerryj89** [Morgul/ui-bootstrap4#67]
### Bug description:
Can't use keyboard for navigating dropdown menu.  Attempting to do so will yield `Uncaught TypeError: Cannot read properties of undefined (reading 'focus')`.

### Link to minimally-working plunker that reproduces the issue:
https://morgul.github.io/ui-bootstrap4 (Dropdown with keyboard nav example)

### Steps to reproduce the issue:
Click on dropdown and use arrow keys

### Version of Angular, UIBS, and Bootstrap

Angular: 1.8.2

UIBS: 3.0.7

Bootstrap: 4

**Comment by Morgul** [comment]
Fixed by #68.
